native/effektio-test/Cargo.toml

```
@@ -14,8 +14,8 @@ features = ["with-mocks"]
 
 [dependencies]
 anyhow = "1"
-env_logger = "0.9.0"
+env_logger = "0.8"
```

why are you dropping to a lower version?

```suggestion
env_logger = "0.9.0"
```

---

`android_logger` is based on `env_logger 0.8`.
If we have 2 versions of `env_logger`, library size would be larger.
I wanted to reduce library size.

---

`android_logger 0.11` is based on `env_logger 0.9`

---

yeah, we should upgrade `android_logger` in that case. ugprading should always be the first choice.

---

In fact, I thought version change of dependency may affect the stability of library.
I just tested `android_logger 0.11` and confirmed it is working well.
https://github.com/effektio/effektio/pull/177/commits/3505714903c9d3698faf299b3d0154f6ce8e479a